org.opentosca.container.core.impl/src/org/opentosca/container/core/impl/plan/PlanInvocationEngine.java

```diff
@@ -104,7 +104,7 @@ public String invokePlan(final CSARID csarID, final QName serviceTemplateId, lon
         }
 
         planEvent.setCSARID(csarID.toString());
-        planEvent.setInputMessageID(ServiceProxy.toscaReferenceMapper.getPlanInputMessageID(csarID, givenPlan.getId()));
+        //planEvent.setInputMessageID(ServiceProxy.toscaReferenceMapper.getPlanInputMessageID(csarID, givenPlan.getId()));
```

Comment:
the removal of that line was a test for https://github.com/OpenTOSCA/container/pull/78, it will be remove in the future anyways, because it is not needed at all (i.e. InputMessageId is not needed at all)